src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -92,7 +96,7 @@ function getBillingStatus(
                 title: translate('subscription.billingBanner.cardExpired.title'),
                 subtitle: translate('subscription.billingBanner.cardExpired.subtitle', {amountOwed}),
                 isError: true,
-                isRetryAvailable: true,
+                isRetryAvailable: !isCurrentCardExpired,
             };
 
         case SubscriptionUtils.PAYMENT_STATUS.CARD_EXPIRE_SOON:
```

Comment:
After reading the [thread](https://github.com/Expensify/App/pull/44268/files#r1662442357) again, Do you think we should add `isRetryAvailable: true` in this case?

Reply:
Like I've said in the thread - what we're returning is not an error (`isError: false`) so there is nothing to retry in that scenario. Its just a message for the user to let him know that his card will be expiring soon.